src/sonic-frr-mgmt-framework/frrcfgd/frrcfgd.py

```diff
@@ -1791,9 +1802,9 @@ class BGPConfigDaemon:
                    (['local_asn', '+local_as_no_prepend',
                      '+local_as_replace_as'],               '{no:no-prefix}neighbor {} local-as {} {:no-prepend} {:replace-as}'),
                    (['admin_status', '+shutdown_message'],  '{no:no-prefix}neighbor {} shutdown {:shutdown-msg}', ['false', 'true']),
-                   ('local_addr',                           '{no:no-prefix}neighbor {} update-source {}'),
+                   ('update_source',                           '{no:no-prefix}neighbor {} update-source {}'),
```

Comment:
This is a backward incompatible change; I would keep the local_addr unless this adds any major value.

Reply:
But that would break "show ip interface" since we don't need to have "update source" for normal BGPs.

Reply:
https://github.com/sonic-net/sonic-utilities/blob/317e649514c9b205849ffb5ea96a6a233e38290c/scripts/ipintutil#L51

For that, we need to make the following changes in the above file not the field name change.

 if 'local_addr' in data[neighbor_ip]:
     local_addr = data[neighbor_ip]['local_addr']
 else:
     local_addr = ""

Reply:
Ok. I will revert it and open an issue to track this fix.

Reply:
https://github.com/sonic-net/sonic-buildimage/issues/19930 is opened for track this change.